packages/muelu/doc/UsersGuide/masterList.xml

```diff
@@ -601,6 +601,15 @@
       <visible>false</visible>
     </parameter>
 
+    <parameter>
+      <name>sync factory timers</name>
```

Comment:
The parameter name is ok, but in general I prefer to avoid abbreviations.  Do you object to spelling the option out, i.e., "synchronize factory timers"?